pkgs/applications/editors/neovim/neovim-gtk.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, wrapGAppsHook4
+, pkg-config
+, gtk4
+, glib
+, pango
+, gdk-pixbuf
+, vte-gtk4
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "neovim-gtk";
+  version = "1.0.4";
+
+  src = fetchFromGitHub {
+    owner = "Lyude";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-inva7pYwOw3bXvFeKZ4aKSQ65iCat5HxM+NME8jN4/I=";
+  };
+
+  cargoHash = "sha256-9eZwCOP4xQtFOieqVRBAdXZrXmzdnae6PexGJ/eCyYc=";
+
+  nativeBuildInputs = [ wrapGAppsHook4 pkg-config glib gdk-pixbuf ];
```

I don't think it's using anything from `glib.bin`, and `gdk-pixbuf` should probably be in `buildInputs` instead

```suggestion
  nativeBuildInputs = [ wrapGAppsHook4 pkg-config ];
```